c/bpxskt.c

```diff
@@ -1041,7 +1053,9 @@ int socketWrite(Socket *socket, const char *buffer, int desiredBytes,
     int bytesWritten = 0;
     status = rs_ssl_write(socket->sslHandle, buffer, desiredBytes, &bytesWritten);
     if (0 != status) {
-      printf("socketWrite: rs_ssl_write failed with status: %d\n", status);
+      if (socketTrace){
+        printf("socketWrite: rs_ssl_write failed with status: %d\n", status);
```

Comment:
returncode & reasoncode are not set yet, but status is. should we return status to the user so that this print is not needed?